src/components/Weather.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div style="position: relative">
+        <div v-if="getweatherimg == '01d' || getweatherimg == '01n'">
+            <img src="../../src/assets/images/weatherimg/jour/beau.png" alt="">
+        </div>
+        <div v-else-if="getweatherimg == '50d'">
+            <img src="../../src/assets/images/weatherimg/jour/brouillard.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '13d'">
+            <img src="../../src/assets/images/weatherimg/jour/neige.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '04d' || getweatherimg == '04n'">
+            <img src="../../src/assets/images/weatherimg/jour/nuage-gris.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '03d' || getweatherimg == '03n'">
+            <img src="../../src/assets/images/weatherimg/jour/nuage.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '11d'">
+            <img src="../../src/assets/images/weatherimg/jour/orage.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '09d'">
+            <img src="../../src/assets/images/weatherimg/jour/pluie.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '02d' || getweatherimg == '02n'">
+            <img src="../../src/assets/images/weatherimg/jour/soleil-nuage.png" alt="" >
+        </div>
+        <div v-else-if="getweatherimg == '10d'">
+            <img src="../../src/assets/images/weatherimg/jour/soleil-pluie.png" alt="" >
+        </div>
+        <div class="weathedescription">
+            <p>{{currentDate()}}</p>
+            <p class="weathedescription-1">{{ getweathedescription }}</p>
+            <p class="weathetemp">{{ getweathertemp }} °C</p>
+        </div>
+    </div>
+</template>
+
+<script>
+
+
+import axios from 'axios';
+import moment from 'moment';
+export default {
+    name:'Weather',
+    data(){
+        return {
+            getweathertemp: undefined,
+            getweatherimg: undefined,
+            getweathedescription: undefined,
+            date: undefined
+        }
+    },
+    methods: {
+    currentDate() {
+      const current = new Date();
+      const dateLocale = current.toLocaleString('fr-FR',{
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric'});
+      return dateLocale;
+    }
```

Comment:
Faut que ce soit un computed, mais c'est mal indenté, il manque une tabulation

```suggestion
      currentDate() {
        const current = new Date();
        const dateLocale = current.toLocaleString('fr-FR',{
              year: 'numeric',
              month: 'long',
              day: 'numeric'});
        return dateLocale;
      }
```